docs/networking/connecting-the-arena-to-a-network.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: connecting-the-arena-to-a-network 
-title: Connecting the Arena to a network
-sidebar_label: Connecting the Arena to a network
+title: Connecting the Diamond 9 or Arena to a network
+sidebar_label: Connecting the Diamond 9 or Arena to a network
 ---
 
-The Arena console is different to the other Titan consoles because it
-has a built-in network switch and an optical network connection
-(optionally 2 optical connections). **This section applies only to the
-Arena console.**
+The Diamond 9 and Arena consoles are different to the other Titan consoles because they
+have a built-in network switch and an optical network connection. 
+**This section applies only to the Diamond 9 and 
+Arena consoles.**
 
 Optical Connections
 -------------------
 
 The optical network connector is a Neutrik opticalCON Duo which uses
```

Comment:
On the D9, it's based on a Quad, with one of the two pairs populated by default. If the 10GbE ethernet card option is fitted, then the second pair is populated with a 10GbE fibre.